relayer/app/cursor/store.go

```diff
@@ -134,13 +141,9 @@ func (s *Store) Insert(
 		existing, err := s.db.Get(ctx, srcVersion.ID, uint32(srcVersion.ConfLevel), destChain, attestOffset)
 		if err != nil {
 			return errors.Wrap(err, "get cursor")
-		} else if !maps.Equal(offsetsByShard, existing.GetStreamOffsetsByShard()) { // For now just log a bug if this happens.
-			log.Error(ctx, "Unexpected existing cursor offset mismatch [BUG]", nil,
-				"src_chain_version", s.network.ChainVersionName(srcVersion),
-				"dest_chain", s.network.ChainName(destChain),
-				"attest_offset", attestOffset,
+		} else if !maps.Equal(offsetsByShard, existing.GetStreamOffsetsByShard()) { // For now just log an error if this happens.
+			log.Error(ctx, "Unexpected existing cursor offset mismatch", nil,
```

Comment:
I wonder if saying it's `unexpected` is still correct?

Reply:
it is indeed incorrect